modules/aws-outposts-load-balancer-clb.adoc

```diff
@@ -0,0 +1,133 @@
+
+* You have access to the cluster using an account with `cluster-admin` permissions.
+
+* You have installed the {oc-first}.
+
+* You have installed the AWS Load Balancer Operator and created the AWS Load Balancer Controller.
+
+.Procedure
+
+. List the nodes on the Outpost by running the following command:
++
+[source,terminal]
+----
+$ oc get nodes
+----
++
+.Example output
+[source,terminal]
+----
+NAME                   STATUS    ROLES     AGE       VERSION
+node1.example.com      Ready     worker    7h        v1.28.5
+node2.example.com      Ready     worker    7h        v1.28.5
+----
+//Does this need some additional filtering to ensure we are only listing Outposts nodes and not the VPC nodes too?
+
+. For each, label the node with a key-value pair by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node_name> <key_name>=<value>
+----
++
+#is there a way to label them all at once in case there are a bunch? Something related to the `--all` parameter here? https://docs.openshift.com/container-platform/4.14/nodes/nodes/nodes-nodes-working.html#nodes-nodes-working-updating_nodes-nodes-working#
+
+. #How does the user find these to edit them? Or is this assuming from new?#
+
+. Edit the Classic Load Balancer `deployment` resource:
++
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: <application_name>
+  namespace: <application_namespace>
+spec:
+  selector:
+    matchLabels:
+      app: <application_name>
+  replicas: 3
+  template:
+    metadata:
+      labels:
+        app: <application_name>
+    spec:
+      containers:
+        - image: openshift/origin-node
+          command:
+           - "/bin/socat"
+          args:
+            - TCP4-LISTEN:8080,reuseaddr,fork
+            - EXEC:'/bin/bash -c \"printf \\\"HTTP/1.0 200 OK\r\n\r\n\\\"; sed -e \\\"/^\r/q\\\"\"'
+          imagePullPolicy: Always
+          name: <application_name>
+          ports:
+            - containerPort: 8080
+----
+
+. Create the `deployment` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_deployment>.yaml
+----
+
+. Edit the Classic Load Balancer `service` resource:
++
+.Example service configuration
+[source,yaml]
+----
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: <application_name>
+  name: <application_name>
+  namespace: <application_namespace>
+  annotations:
+    service.beta.kubernetes.io/aws-load-balancer-subnets: <aws_subnet>
+    service.beta.kubernetes.io/aws-load-balancer-target-node-labels: <key_name>=<value> # <1>
+spec:
+  ports:
+  - name: http
+    port: 80
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: <application_name>
+  type: LoadBalancer
+----
+<1> Specify the key-value pair that matches the pair in the node label.
+
+. Create the `service` resource CR by running the following command:
++
+[source,terminal]
+----
+$ oc create -f <clb_service>.yaml
+----
+
+.Verification
+
+* #To add.#
```

Comment:
Refer to the last step [Verification](https://docs.openshift.com/container-platform/4.14/networking/aws_load_balancer_operator/create-instance-aws-load-balancer-controller.html ) 
if we check from aws console, click "Load Balancers -> Target instances" we can see only the labeled instances will show <img width="1523" alt="Screenshot 2024-02-21 at 17 58 17" src="https://github.com/openshift/openshift-docs/assets/37139281/28ddae2c-1937-474b-9a49-19097cb4fd76">.